pressbooks/pressbooks-book
# Add author for each individual chapter in webbook's TOC

Issue #784

This PR adds information about the authors for each chapter in the webbook's TOC.

### How to test

1. Go to the desired book and add author or authors to individual chapters (front-matter, chapters, back-matter)
2. Visit the book's home page and see if the TOC renders properly
3. Visit a specific chapter and see if the TOC renders properly in the reading interface
4. Include sections to chapters and enable two-level toc under Appearance > Theme Options
5. Visit the book's home page and see if the TOC renders properly
